css/index.css

```diff
@@ -65,7 +66,33 @@ h1, h2, h3, h4, h5 {
 p {
     line-height: 1.4;
 }
+.top-nav-bar {
+    display:flex;
```

Comment:
You css is very good, my only feedback is to remove unecessary spaces and add more comments so you can follow the flow of your code better